as/plugins/org.jboss.ide.eclipse.as.management.wildfly8/.classpath

```diff
@@ -2,17 +2,17 @@
 <classpath>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.6"/>
```

Comment:
Denis:

Why do these classpath entries need to be exported? They shouldn't be. The jars in the wf9 plugin should remain 100% internal and should not be exported to anyone at all.